arangod/VocBase/LogicalCollection.h

```diff
@@ -365,7 +365,7 @@ class LogicalCollection : public LogicalDataSource {
    * Enterprise only method. See enterprise code for implementation
    * Community has a dummy stub.
    */
-  std::string createSmartToSatKey(arangodb::velocypack::Slice input);
+  std::string createKeyBetweenSmartAndSat(arangodb::velocypack::Slice edgeSlice) const;
```

Comment:
```suggestion
  std::string createKeyBetweenSmartAndSat(arangodb::velocypack::Slice edgeSlice);
```

Reply:
using keyGenerator is not allowed to be const. As @mchacki told me once, that the key generator uses / increments ticks to be able to create unique keys.